dotnet/src/SemanticKernel.Abstractions/AI/Embeddings/ITextEmbeddingGenerationService.cs

```diff
@@ -7,5 +7,5 @@ namespace Microsoft.SemanticKernel.Embeddings;
 /// <summary>
 /// Represents a generator of text embeddings of type <c>float</c>.
 /// </summary>
-[Experimental("SKEXP0001")]
+// [Experimental("SKEXP0001")]
```

Comment:
Any service except IChatCompletionService and ITextGenerationService are marked as Experimental by the Semantic Kernel so I suppressed them in order to build the project when I wrote out TextEmbedding. I'll be adding the Experimental tag to my implementations of TextEmbedding and TextToImage so that so services build later and uncomment the Semantic Kernel files so that they are untouched. They are reverted to original state on this PR